webhooks.md

```diff
@@ -0,0 +1,160 @@
+# PayrollHero Webhooks
+
+PayrollHero has a webhook system available to notify about various events in an employee's lifecycle.
+
+## Data Objects
+
+The Webhook Interface makes use of standardized payload objects to make sure that the responses are easily handled.
+
+These are:
+
+### Employee Object
+
+```json
+{
+    "id": 1,
+    "account_id": 2,
+    "phone_numbers": [
+      {
+        "label": "main",
+        "number": "+1 123-223-3343"
+      }
+    ],
+    "fullname": "John Doe"
+}
+```
+
+### Shift Object
+```json
+{
+    "clock_in_time": "2016-07-15T19:28:25+00:00",
+    "clock_out_time": "2016-07-15T19:28:25+00:00",
+    "earliest_clock_in_time": "2016-07-15T19:28:25+00:00",
```

Comment:
But there isn't any "allowed to clock in" time anymore.  The multiple rules in PH allow for lots of configurations of this.  No customer ever wanted to "deny" people clocking in earlier than that anyhow.  There's a separate reminder, and then "missing" system which key on the configuration set in payrollhero.